src/scripts/main.js

```diff
@@ -1,3 +1,63 @@
 'use strict';
 
-// write code here
+const appendRowButton = document.querySelector('.append-row');
+const removeRowButton = document.querySelector('.remove-row');
+const appendColumnButton = document.querySelector('.append-column');
+const removeColumnButton = document.querySelector('.remove-column');
+const tableBody = document.querySelector('.field > tbody');
+
+appendColumnButton.addEventListener('click', (e) => {
+  const rows = [...document.querySelector('.field > tbody').children];
```

Comment:
I was wrong about creating this variable outside the function. So you did good job here!